### ISSUE TITLE
AWS Lightsail deploy fails

### ISSUE THREAD

**Issue by defunctio** [trailofbits/algo#1179]
**Describe the bug**
Region is undefined and user is not asked for input.

```
TASK [cloud-lightsail : Get regions] ***********************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "The lightsail_region_facts module requires a region and none was found in configuration, environment variables or module parameters"}
```

**To Reproduce**

Deploy to AWS Lightsail

**Expected behavior**
that it works

**Additional context**

**Full log**

```
TASK [cloud-lightsail : set_fact] **************************************************************************************************
ok: [localhost]

TASK [cloud-lightsail : Get regions] ***********************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "The lightsail_region_facts module requires a region and none was found in configuration, environment variables or module parameters"}

TASK [cloud-lightsail : debug] *****************************************************************************************************
ok: [localhost] => {
    "fail_hint": [
        "Sorry, but something went wrong!",
        "Please check the troubleshooting guide.",
        "https://trailofbits.github.io/algo/troubleshooting.html"
    ]
}

TASK [cloud-lightsail : fail] ******************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed as requested from task"}

PLAY RECAP *************************************************************************************************************************
localhost                  : ok=19   changed=3    unreachable=0    failed=2
```

**Comment by nframi43** [comment]
In config.cfg under Lightsail, change the size from "nano_1_0" to "nano_2_0"

**Comment by Jiraffe7** [comment]
@nframi43 tried that but it doesn't work, still getting the same error.

A workaround was to set a fact called `region: "<your-desired-region-here>"` in `input.yml`

Not a solution, but I managed to get the script working.

**Comment by jackivanov** [comment]
Hm, I can't reproduce on my test environment. @defunctio show the output of `pip freeze` please

**Comment by axiomdata** [comment]
@jackivanov I'm getting the same error as @defunctio. Here is my output from `pip freeze` .

adal==1.2.0
ansible==2.5.2
antlr4-python3-runtime==4.7.1
apache-libcloud==2.3.0
appdirs==1.4.3
applicationinsights==0.11.7
argcomplete==1.9.4
asn1crypto==0.24.0
azure-cli-core==2.0.49
azure-cli-nspkg==3.0.3
azure-cli-telemetry==1.0.0
azure-common==1.1.16
azure-mgmt-batch==4.1.0
azure-mgmt-compute==2.1.0
azure-mgmt-containerinstance==1.2.1
azure-mgmt-containerregistry==2.3.0
azure-mgmt-containerservice==2.0.0
azure-mgmt-dns==1.2.0
azure-mgmt-keyvault==0.40.0
azure-mgmt-network==1.7.1
azure-mgmt-nspkg==3.0.2
azure-mgmt-rdbms==0.2.0rc1
azure-mgmt-resource==1.2.2
azure-mgmt-sql==0.7.1
azure-mgmt-storage==1.5.0
azure-mgmt-web==0.32.0
azure-nspkg==3.0.2
azure-storage==0.35.1
bcrypt==3.1.4
boto==2.49.0
boto3==1.9.33
botocore==1.12.33
certifi==2018.10.15
cffi==1.11.5
chardet==3.0.4
colorama==0.4.0
configparser==3.5.0
cryptography==2.3.1
decorator==4.3.0
docutils==0.14
dogpile.cache==0.6.7
dopy==0.3.5
entrypoints==0.2.3
enum34==1.1.6
futures==3.2.0
humanfriendly==4.17
idna==2.7
ipaddress==1.0.22
iso8601==0.1.12
isodate==0.6.0
Jinja2==2.8
jmespath==0.9.3
jsonpatch==1.23
jsonpointer==2.0
keyring==15.1.0
keystoneauth1==3.11.1
knack==0.4.4
MarkupSafe==1.0
monotonic==1.5
msrest==0.6.1
msrestazure==0.4.34
munch==2.3.2
netifaces==0.10.7
oauthlib==2.1.0
openstacksdk==0.18.1
os-client-config==1.31.2
os-service-types==1.3.0
packaging==18.0
paramiko==2.4.2
pbr==5.1.0
pkg-resources==0.0.0
portalocker==1.2.1
pyasn1==0.4.4
pycparser==2.19
pycrypto==2.6.1
Pygments==2.2.0
PyJWT==1.6.4
PyNaCl==1.3.0
pyOpenSSL==18.0.0
pyparsing==2.2.2
python-dateutil==2.7.4
PyYAML==3.13
requests==2.20.0
requests-oauthlib==1.0.0
requestsexceptions==1.4.0
s3transfer==0.1.13
SecretStorage==2.3.1
segno==0.2.8
shade==1.30.0
six==1.11.0
stevedore==1.30.0
tabulate==0.8.2
typing==3.6.6
urllib3==1.24

**Comment by digeratus** [comment]
The PR fix in #1182 worked for me.
